src/pages/home/_trade-the-way-you-like.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import styled from 'styled-components'
 import { graphql, useStaticQuery } from 'gatsby'
-import PropTypes from 'prop-types'
 import { Header, Text, QueryImage } from 'components/elements'
 import { localize } from 'components/localization'
 import { SectionContainer } from 'components/containers'
-import { OtherPlatform } from 'components/custom/other-platforms.js'
+import { OtherPlatform } from 'components/custom/other-platforms'
+
+type IsPpcRedirectProps = {
```

Comment:
```suggestion
type TradeTheWayYouLikeProps = {
```
Shouldn't the common convention be `ComponentNameProps`

Reply:
Oh yeah my bad on that, will fix it.